scripts/build/Jenkins/Jenkinsfile

```diff
@@ -594,6 +594,7 @@ try {
                     // repositoryName is the full repository name
                     repositoryName = (repositoryUrl =~ /https:\/\/github.com\/(.*)\.git/)[0][1]
                     env.REPOSITORY_NAME = repositoryName
+                    env.REPOSITORY_URL = repositoryUrl
```

Comment:
`REPOSITORY_NAME` may be a better one for what you want

Reply:
great suggestion! fixed 👍